src/main/resources/io/fabric8/elasticsearch/plugin/kibana/applications_mapping.json

```diff
@@ -0,0 +1,6 @@
+{
+   "title":"$TITLE$",
+   "timeFieldName":"time",
```

Comment:
right, but when we call the 'create' method in KibanaSeed, we pass in a 'time_field_name' string variable. Why would we not use the value passed in instead of hardcoding "time" here?